assets/styles.scss

```diff
@@ -28,3 +28,9 @@ body {
     margin-bottom: 16px;
   }
 }
+
+body,
+html,
+.d-md-flex {
```

Comment:
We generally don't like to write custom CSS for styles that come from Primer (which `.d-md-flex` does) : https://styleguide.github.com/primer/utilities/flexbox/

Instead, you could use the `.height-full` utility class on the element that you want to have `height: 100%;` on. (Docs: https://styleguide.github.com/primer/utilities/layout/#width-and-height)